pkg/rpc/context.go

```diff
@@ -901,11 +911,9 @@ func (d delayingConn) Read(b []byte) (n int, err error) {
 		// If we somehow don't get our expected magic, throw an error.
 		if hdr.Magic != magic {
 			panic(errors.New("didn't get expected magic bytes header"))
-			// TODO (rohany): I can't get this to work. I suspect that the problem
```

Comment:
i was confused about this TODO. I hit it easily when I do `\demo shutdown <node>`

Reply:
The TODO was about the thing below. You're right that shutdown is broken... we can fix that separately.